### PR TITLE
Add support for actions on attachments (message buttons)

### PR DESCRIPTION
**Pull request by ghost**
As defined in https://api.slack.com/docs/message-buttons.

### CODE REVIEW

File: src/SlackConnector/Models/SlackAttachmentActionStyle.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace SlackConnector.Models
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum SlackAttachmentActionStyle
```
Comment (Workshop2) [inline src/SlackConnector/Models/SlackAttachmentActionStyle.cs:8]: To follow naming conventions, can you add "Type" onto the end please

File: src/SlackConnector/Models/SlackAttachmentActionStyle.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        @default,
```
Comment (Workshop2) [inline src/SlackConnector/Models/SlackAttachmentActionStyle.cs]: Again, to follow naming standards can you make these values CamelCase and maybe use a converter to lowercase if you need lowercase (or use something like `[EnumMember(Value = "default")]`)

**Comment by Workshop2** [comment]
Everything looks fine except for a couple of naming bits. If you are able to resolve those then I will merge and publish ASAP :+1:

File: src/SlackConnector/Models/SlackAttachmentActionStyle.cs (hunk 1 above)
Comment (ghost) [inline src/SlackConnector/Models/SlackAttachmentActionStyle.cs:8]: I have followed the implicit convention of naming the enum by using the 'parent' class name suffixed with property name, as on SlackChatHubType (used on SlackChatHub class' Type property).

File: src/SlackConnector/Models/SlackAttachmentActionStyle.cs (hunk 2 above)
Comment (ghost) [inline src/SlackConnector/Models/SlackAttachmentActionStyle.cs]: I will fix these.

**Comment by Workshop2** [comment]
Thanks for the help @andre-takenet

**Pull request merged**